### PR TITLE
DOCS-2355 log forwarding ph

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/managing-via-the-ops-center.adoc [hunk 1]
```diff
@@ -87,7 +87,7 @@ image::ops-center-settings.png[]
 |Key | Description
 |`name` | Specifies the name of the log forwarding rule.
 |`address` | Specifies the endpoint and port to forward the log data
-|`protocol` | Specifies the protocol to send the data to. Supported protocols are TCP or UDP.
+|`protocol` | Specifies the protocol to send the data to. Supported protocols are TCP or UDP. Defaults to `tcp`.
```
Comment (dhirsch1) [inline modules/ROOT/pages/managing-via-the-ops-center.adoc]: Is this a list that has the values preset? If it is a field, it is not clear what the user enters for UDP. Is it udp?

**Review by dhirsch1 — APPROVED** [review]
Approved with one comment to address.

File: modules/ROOT/pages/managing-via-the-ops-center.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/managing-via-the-ops-center.adoc]: Good point - I will clarify that. Thank you!

**Comment by paigehargrave** [comment]
Closing this in favor of #65. Install topic updates in #64.

**Pull request closed without merging**